apps/test-blinky/src/TutorialServo.cpp

```diff
@@ -0,0 +1,42 @@
+//
+//  TutorialServo.cpp
+//  
+//
+//  Created by Ari Wasch on 9/18/20.
+//
+
+#include "TutorialServo.h"
+#include "mbed.h"
+
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2){
+    m_servoPwmOut(PA_1)
+    m_servoPwmOut.period(0.020);
+    m_servoRangeInDegrees = servoRangeInDegrees
+    m_minPulsewidthInMs = minPulsewidthInMs
+    m_maxPulsewidthInMs = maxPulsewidthInMs
+
+}
+ 
+void TutorialServo::setAngleRangeInDegrees(float degrees) {
+    m_servoRangeInDegrees = servoRangeInDegrees
+}
+void TutorialServo::setPulsewidthRangeInMs(float minPulsewidthMs, float maxPulsewidthMs) {
+    m_minPulsewidthInMs = minPulsewidthInMs
+    m_maxPulsewidthInMs = maxPulsewidthInMs
+
+}
+
+void setPositionInDegrees(float degrees) {
```

Comment:
currently this is not a member function. this doesnt define a member function of TutorialServo like the above do! dont forget to scope!